### DISCUSSION TITLE
Migrate my project from Userkit to Devkit?

### DISCUSSION THREAD

**Discussion by olifurz** [CosmosOS/Cosmos#2094] [Q&A]
Hi,

I've got a project I've been working on, I'd like to know how do I install the dev kit, and how do I migrate my project from the userkit to the devkit without a load of effort?

**Comment by selkij** [comment]
just install the devkit
you normally have nothing elses to do

**Reply by olifurz** [reply]
How do I install it though? I find the documentation on it really confusing.

**Reply by selkij** [reply]
just run the install-vs2019.bat

**Reply by quajak** [reply]
What part is confusing?

**Reply by olifurz** [reply]
I get the error "Windows cannot find source\Cosmos.Build.Builder\bin\Debug\Cosmos.Build.Builder.exe"

**Reply by quajak** [reply]
Have you checked https://github.com/CosmosOS/Cosmos/issues/1612 and #1892 ? If they dont work please open a new issue, we should be able to fix that error.

**Reply by olifurz** [reply]
@quajak I'm confused by where to place all the zips?
![wevqwqrweq](https://user-images.githubusercontent.com/97028115/152701963-9d0a73d4-b6b6-4c2a-8b31-aa9ca37fbe11.png)

**Reply by olifurz** [reply]
I'm confused by the "C:\source\CosmosOS" part, as I do not have a folder called Cosmos OS, only repos. Inside repos there's a folder called just Cosmos containing just a .git folder, and another folder called CosmosGit in repos.

**Reply by quajak** [reply]
The zips should be unzipped into the same parent folder. The parent folder does not need to be called CosmosOS.
